src/components/Calculator/index.jsx

```diff
@@ -1,69 +1,83 @@
 import React from 'react';
 import Display from '../Display';
 import Number from '../Number';
-import Operator from '../Operator';
-import Result from '../Result';
+import OperationEditor from '../OperationEditor';
 
-const Calculator = () => (
-    <div align="center">
-        <Display
-            number={5}
-        />
-        <table>
-            <tr>
-                <td>
-                    <Number displayNumber={7}/>
-                </td>
-                <td>
-                    <Number displayNumber={8}/>
-                </td>
-                <td>
-                    <Number displayNumber={9}/>
-                </td>
-            </tr>
-            <tr>
-                <td>
-                    <Number displayNumber={4}/>
-                </td>
-                <td>
-                    <Number displayNumber={5}/>
-                </td>
-                <td>
-                    <Number displayNumber={6}/>
-                </td>
-            </tr>
-            <tr>
-                <td>
-                    <Number displayNumber={1}/>
-                </td>
-                <td>
-                    <Number displayNumber={2}/>
-                </td>
-                <td>
-                    <Number displayNumber={3}/>
-                </td>
-            </tr>
-            <tr>
-                <td>
-                    <Operator operator={'+'}/>
-                </td>
-                <td>
-                    <Operator operator={'-'}/>
-                </td>
-                <td>
-                    <Operator operator={'*'}/>
-                </td>
-                <td>
-                    <Operator operator={'/'}/>
-                </td>
-            </tr>
-            <tr>
-                <td></td>
-                <td><Result/></td>
-                <td></td>
-            </tr>
-        </table>
-    </div>
-);
+const Calculator = () => {
+    const[currentNumber, setCurrentNumber] = React.useState('');
```

Comment:
If this is a number, I think the best default value could be 0.

Reply:
I am using a string because I will do a concatenation.

You clic 2, you have 2
You clic 3, you have 23

Do you recommend me other way to manage the number?